### PR TITLE
docs: update QA with push notification test sequence

### PR DESCRIPTION
**Pull request by alexruzenhack**
### Acceptance Criteria
- update QA with the push notification test sequence

### Security Checklist
- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

**Review by andreabadesso — CHANGES_REQUESTED** [review]

File: src/sagas/wallet.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ import { eventChannel } from 'redux-saga';
 import { getUniqueId } from 'react-native-device-info';
 import { t } from 'ttag';
 import { get } from 'lodash';
+import AsyncStorage from '@react-native-async-storage/async-storage';
```
Comment (pedroferreira1) [inline src/sagas/wallet.js:36]: What's this?

File: QA.md [hunk 2]
```diff
@@ -123,3 +123,92 @@
...
+1. Turn on the `Show amounts on notification`
+1. Turn off the `Enabled Push Notification`
+1. Try to turn on the `Show amounts on notification`, it should not be enabled
+1. Turn on the `Enabled Push Notification`
+1. Turn off the `Show amounts on notification`
+1. Turn off the `Enabled Push Notification`
+1. **Go to the Settings page**
+1. Remove your `deviceId` from the unleash **`push-notification.rollout`** feature toggle
+1. Wait until the Push Notification item disappears from the Settings page
+1. Send HTR to this new wallet, wait 3-5 min
```
[the remark this answers is not in the document]
Comment (alexruzenhack) [reply QA.md]: Sometimes the firebase takes a time to deliver the message. Sometimes the wallet-daemon takes a time to deliver the transaction to the lambda "onNewTxRequest".

File: src/sagas/wallet.js (hunk 1 above)
Comment (alexruzenhack) [reply src/sagas/wallet.js]: A lack of attention. I removed this unused import in [9f3ff55](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/248/commits/9f3ff558164e3d6ca930ad3e54256886e5c6fb69).

File: QA.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (andreabadesso) [reply QA.md]: I understand we can't control firebase, but we should definetely debug why the wallet-daemon takes time to deliver the transaction, it should be close to instant as we are syncing the mempool

[the remark this answers is not in the document]
Comment (alexruzenhack) [reply QA.md]: Mmmm... I don't have a statistic about it, but I record in the tests that 99% of the notification comes almost instantaneous indeed.

**Review by andreabadesso — APPROVED** [review]

File: QA.md [hunk 3]
```diff
@@ -123,3 +123,268 @@
...
+    1. Wait until the modal with tx details open
+        1. The `HTR - HATHOR` name should be in the primary color (purple)
+        1. The `TN1 - Test Nft` name should be in the primary color (purple)
+        1. The `TN1 - Test Nft` amount should be integer
+    1. Click on the `TN1 - Test Nft` item
+        1. The **Balance** page for `TN1` token should open
+1. **Send token to self**
+    1. Wait 3-5 min
+    1. You should **not** receive a notification of new transaction
+1. Test open the wallet 2 weeks later
```
Comment (pedroferreira1) [inline QA.md:351]: We usually run the QA after sending to the stores for beta versions. You should mark this item here to be done only if it's running locally

File: QA.md [hunk 4]
```diff
@@ -123,3 +123,268 @@
...
+    1. Click on the `TTT - Test Token Test` item
+        1. Nothing should happen
+    1. Click on the `TN1 - Test Nft` item
+        1. Nothing should happen
+    1. Click on the `TNT - Test Nft Test` item
+        1. Nothing should happen
+    1. Click on the `HTR - HATHOR` item
+        1. The **Balance** page should open
+1. **Register `TTT` token and send 2 tokens after turn on `Show amounts on notification` option**
+    > WARNING: Not possible using `wallet-service` due to a validation that consults the fullnode.
```
Comment (pedroferreira1) [inline QA.md:314]: Only in the testnet, right?

**Review by pedroferreira1 — APPROVED** [review]

File: QA.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (pedroferreira1) [reply QA.md]: Maybe we could say only "wait until the notification arrives" and for the steps that we should not receive the notification just say "guarantee you won't receive any notifications for this tx". We can guarantee that if we send different amounts for each tx, so the first you send 0.01, second 0.02, ... then you know where this push notification came from. (or just check the tx id)

File: QA.md (hunk 4 above)
Comment (alexruzenhack) [reply QA.md:314]: Yes, only in testnet. I will add this.

File: QA.md (hunk 3 above)
Comment (alexruzenhack) [reply QA.md:351]: [6638db6](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/248/commits/6638db6d32dc3a51876cbd3c8b180181fd321dbc)

File: QA.md (hunk 4 above)
Comment (alexruzenhack) [reply QA.md]: [6638db6](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/248/commits/6638db6d32dc3a51876cbd3c8b180181fd321dbc)

File: QA.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexruzenhack) [reply QA.md]: [f0ffb96](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/248/commits/f0ffb96634abdda16d64bd8df9481e7a816aa77d)

**Pull request merged**